libs/sdk-backend-bear/src/backend/workspace/metadata/index.ts

```diff
@@ -220,4 +224,75 @@ export class BearWorkspaceMetadata implements IWorkspaceMetadata {
             }),
         );
     }
+
+    public getUrisByIdentifiers = async (identifiers: Identifier[]): Promise<IUriIdentifierObject[]> => {
+        const uriIdentifierPairs = await this.authCall((sdk) =>
+            sdk.md.getUrisFromIdentifiers(this.workspace, identifiers),
+        );
+        return uriIdentifierPairs.map(convertUriIdentifierPair);
+    };
```

Comment:
we have a function in `utils/api` for this: objRefsToUris. on input the services should be accepting the opaque objRef, then use this function to do the conversion.

Reply:
I have removed the function.